### ISSUE TITLE
Ablation study: learning algo ingredients

### ISSUE THREAD

**Issue by mensch72** [mensch72/empo#90]
Find out what contribution to convergence the following learning algorithm ingredients make:
- gradient clipping
- weight decay
- adaptive exploration
- warm-up and ramp-up phases
- ... (?)
